integrated_channels/integrated_channel/learner_data.py

```diff
@@ -134,6 +133,34 @@ def collect_learner_data(self):
                 is_passing=is_passing,
             )
 
+    def _should_skip_enrollment(self, enterprise_enrollment):
+        """
+        Specifies whether to skip data collection of the enterprise enrollment due to any configuration reason.
+
+        :param enterprise_enrollment: The enterprise enrollment whose configuration is to be checked against.
+        :return: True if we should skip, False otherwise.
+        """
+        return any([
+            not enterprise_enrollment.consent_available,
```

Comment:
I say in the docstring that we skip data collection for an enterprise enrollment due to configuration reasons: since `consent_available` is a property (i.e. dependent on some configuration), it's one of the things we check for to determine the skip. But I will try to word the docstring a little better, yes.